geopandas/tests/test_pandas_methods.py

```diff
@@ -306,7 +307,14 @@ def test_numerical_operations(s, df):
 
     # df methods ignore the geometry column
     exp = pd.Series([3, 4], index=["value1", "value2"])
-    assert_series_equal(df.sum(), exp)
+    msg = "Dropping of nuisance columns in DataFrame reductions"
+    with warnings.catch_warnings():
+        warnings.filterwarnings(
+            "ignore",
+            category=FutureWarning,
```

Comment:
For this kind of cases, we could also use

```
@pytest.mark.filterwarnings(
    "ignore:Dropping of nuisance columns in DataFrame reductions:FutureWarning"
)
```

That's a pattern we already use elsewhere (and I think it is fine to use here since the test is basically doing only a single thing)

Reply:
Thanks, that's much more succint.

I'll mark this as ready as well - originally I was going to look at dealing with more warnings but I'll leave to these for now.